packages/schematics/angular/component/schema.json

```diff
@@ -130,6 +130,11 @@
       "type": "boolean",
       "default": false,
       "description": "The declaring NgModule exports this component."
+    },
+    "exportDefault": {
```

Comment:
Maybe we should name this `exportAsDefault` since it clearly communicates that the component will be exported as a default export.

@clydin/@dgp1130 thoughts?

Reply:
While more explicit in regards to intention, I think `--export-as-default` is fairly verbose from a usability perspective.  I do not have a strong opinion either way but would probably prefer `--export-default`.

Reply:
In that case let’s go with `--export-default`.